internal/config/yaml_parser_test.go

```diff
@@ -16,253 +16,234 @@ package config
 
-	AssertEq(nil, err)
-	AssertNe(nil, mountConfig)
-	ExpectEq(true, mountConfig.WriteConfig.CreateEmptyFile)
-	ExpectEq(ERROR, mountConfig.LogConfig.Severity)
-	ExpectEq("/tmp/logfile.json", mountConfig.LogConfig.FilePath)
-	ExpectEq("text", mountConfig.LogConfig.Format)
+	assert.NoError(t.T(), err)
+	assert.NotNil(t.T(), mountConfig)
+	assert.True(t.T(), mountConfig.WriteConfig.CreateEmptyFile)
+	assert.Equal(t.T(), ERROR, mountConfig.LogConfig.Severity)
+	assert.Equal(t.T(), "/tmp/logfile.json", mountConfig.LogConfig.FilePath)
+	assert.Equal(t.T(), "text", mountConfig.LogConfig.Format)
 
 	// log-rotate config
-	ExpectEq(100, mountConfig.LogConfig.LogRotateConfig.MaxFileSizeMB)
-	ExpectEq(5, mountConfig.LogConfig.LogRotateConfig.BackupFileCount)
-	ExpectEq(false, mountConfig.LogConfig.LogRotateConfig.Compress)
+	assert.Equal(t.T(), 100, mountConfig.LogConfig.LogRotateConfig.MaxFileSizeMB)
+	assert.Equal(t.T(), 5, mountConfig.LogConfig.LogRotateConfig.BackupFileCount)
+	assert.False(t.T(), mountConfig.LogConfig.LogRotateConfig.Compress)
 
 	// metadata-cache config
-	ExpectEq(5, mountConfig.MetadataCacheConfig.TtlInSeconds)
-	ExpectEq(1, mountConfig.MetadataCacheConfig.TypeCacheMaxSizeMB)
-	ExpectEq(3, mountConfig.MetadataCacheConfig.StatCacheMaxSizeMB)
+	assert.Equal(t.T(), int64(5), mountConfig.MetadataCacheConfig.TtlInSeconds)
+	assert.Equal(t.T(), 1, mountConfig.MetadataCacheConfig.TypeCacheMaxSizeMB)
+	assert.Equal(t.T(), int64(3), mountConfig.MetadataCacheConfig.StatCacheMaxSizeMB)
 
 	// list config
-	ExpectEq(true, mountConfig.ListConfig.EnableEmptyManagedFolders)
+	assert.True(t.T(), mountConfig.ListConfig.EnableEmptyManagedFolders)
 
 	// auth config
-	ExpectEq(true, mountConfig.AuthConfig.AnonymousAccess)
+	assert.True(t.T(), mountConfig.AuthConfig.AnonymousAccess)
 
 	// enable-hns
-	ExpectEq(true, mountConfig.EnableHNS)
+	assert.Equal(t.T(), true, bool(mountConfig.EnableHNS))
 
 	// file-system config
-	ExpectTrue(mountConfig.FileSystemConfig.IgnoreInterrupts)
+	assert.True(t.T(), mountConfig.FileSystemConfig.IgnoreInterrupts)
 }
 
 func (t *YamlParserTest) TestReadConfigFile_InvalidLogConfig() {
 	_, err := ParseConfigFile("testdata/invalid_log_config.yaml")
 
-	AssertNe(nil, err)
-	AssertTrue(strings.Contains(err.Error(),
-		fmt.Sprintf(parseConfigFileErrMsgFormat, "log severity should be one of [trace, debug, info, warning, error, off]")))
+	assert.ErrorContains(t.T(), err, fmt.Sprintf(parseConfigFileErrMsgFormat, "log severity should be one of [trace, debug, info, warning, error, off]"))
 }
 
 func (t *YamlParserTest) TestReadConfigFile_InvalidLogRotateConfig1() {
 	_, err := ParseConfigFile("testdata/invalid_log_rotate_config_1.yaml")
 
-	AssertNe(nil, err)
-	AssertTrue(strings.Contains(err.Error(),
-		fmt.Sprintf(parseConfigFileErrMsgFormat, "max-file-size-mb should be atleast 1")))
+	assert.ErrorContains(t.T(), err, fmt.Sprintf(parseConfigFileErrMsgFormat, "max-file-size-mb should be atleast 1"))
```

Comment:
Nit: You have corrected above (t, expected, actual) order. If so can you do it the same here also?

Reply:
The above order is correct for ErrorContains: https://pkg.go.dev/github.com/stretchr/testify/assert#ErrorContains